charts/brigade/templates/gateway-generic-ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.genericGateway.ingress.enabled -}}
+{{- $serviceName := include "brigade.genericGateway.fullname" . -}}
+{{- $servicePort := .Values.genericGateway.service.externalPort -}}
+{{- $paths := .Values.genericGateway.ingress.paths -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $serviceName }}
+  labels:
+    app.kubernetes.io/name: {{ template "brigade.genericGateway.fullname" . }}
```

Comment:
Q: this could/should also be represented as `{{ $serviceName }}`, right?

Reply:
I think it could - I took most of the template from the API ingress, so I think it would make sense to be consistent - if it makes sense, we can change this across all ingresses.